src/kontrol/prove.py

```diff
@@ -675,14 +665,9 @@ def _final_cterm(
     return final_cterm
 
 
-def _final_term(empty_config: KInner, contract_name: str, is_test: bool, use_init_code: bool = False) -> KInner:
-    program = (
-        KEVM.init_bytecode(KApply(f'contract_{contract_name}'))
-        if use_init_code
-        else KEVM.bin_runtime(KApply(f'contract_{contract_name}'))
-    )
+def _final_term(empty_config: KInner, program: KInner, is_test: bool, is_setup: bool) -> KInner:
     post_account_cell = KEVM.account_cell(
-        Foundry.address_TEST_CONTRACT() if is_test else Foundry.address_TEST_SYMBOLIC(),
+        Foundry.address_TEST_CONTRACT(),
```

Comment:
Can you also remove [`Foundry.address_TEST_SYMBOLIC()` ](https://github.com/runtimeverification/kontrol/blob/master/src/kontrol/foundry.py#L343)from foundry.py since it is not used anymore?

Reply:
Done.